build/build.sh

```diff
@@ -21,13 +14,7 @@ cd "$BOOST"
 ./b2 "$PROJECT_DIR/test" -q "$@"
 
 cd "$PROJECT_DIR"
-export BOOSTBOOK_DIR="$PROJECT_DIR/bin/turtle/boostbook"
-copy -r "$BOOST"/tools/boostbook/xsl "$BOOSTBOOK_DIR"
-copy -r "$BOOST"/tools/boostbook/dtd "$BOOSTBOOK_DIR"
-copy -r build/boostbook/* "$BOOSTBOOK_DIR"
-copy "$BOOST"/doc/src/boostbook.css doc/html
-copy "$BOOST"/doc/src/images/*.png doc/html/images
-copy "$BOOST"/doc/src/images/callouts/*.png doc/html/images/callouts
+scripts/build_doc.sh "$@"
 
 cd "$BOOST"
-./b2 "$PROJECT_DIR/doc" -q "$@"
+./b2 "$PROJECT_DIR/doc//mock_examples" -q "$@"
```

Comment:
Why the two `/`?

Reply:
That's b2: go into that folder and build that target

Reply:
I meant why `"$PROJECT_DIR/doc//mock_examples"` instead of `"$PROJECT_DIR/doc/mock_examples"`?

Reply:
B2 uses a double slash to delimit folder and target 

mock_examples is a target in the folder $PROJECT_DIR/doc. See the jamfile

Reply:
Oh right, thanks!